### ISSUE TITLE
Issue #1 - just a simple thing

### ISSUE THREAD

**Issue by dananichev** [dananichev/ava-jspm-example#1]
Hey @sindresorhus!

Apparently there is no notifications for closed issues and i can't reopen this one by myself 
https://github.com/sindresorhus/ava/issues/131

Please read the comment https://github.com/sindresorhus/ava/issues/131#issuecomment-154083207

**Comment by sindresorhus** [comment]
I still don't get what you are trying to achieve with this and why? Test files are run through Babel. Your code should be something Node.js can run, meaning you'll need to transpile it before testing if you want to use ES2015+ features.

**Comment by dananichev** [comment]
@sindresorhus it's just an example of the application. 

Let's suppose we have a lot of modules with external dependencies (in this example - underscore). This dependencies managed by JSPM (JSPM provides module loader -- via SystemJS). 

With JSPM ```import * as utils from 'underscore.strings';``` will be resolved as ```jspm_packages/npm/undesrcore.strings/[*]```. 

W/o JSPM this import will be resolved as ```./underscore.strings``` and wont work. 

This command:
```
jspm dl-loader --babel
```
is used to specify babel as transpiler and nothing more (for example, you can use  ```traceur```)

Looking back at karma and jasmine they are using workflow similar to this: 

* Before all:
    * test runner should load specified module loader (SystemJS in this case)
    * test runner should load SystemJS config
* Test runner should load spec files and execute them

Selection of Module Loader and SystemJS config both are based on test runner config.

**TL; DR:**
I wanted to know how to specify some kind of config for cases similar to this one (custom ES6 Module Loader and so on).

PS. Sorry for my english. It sucks.

**Comment by sindresorhus** [comment]
Testing should be decoupled from all the JSPM stuff. Just compile it before testing:

```js
$ jspm-compile-your-code-to-a-commonjs-bundle && ava test.js
```

> custom ES6 Module Loader

There's nothing called the ES6 Module Loader. There isn't even an official spec for the module loader yet.

-

Example: https://github.com/sindresorhus/pageres/blob/28dd9b3004c05470202f02e0ac0d0ee148a63768/package.json#L16

See how it compiles the library before using it in the tests.

**Comment by dananichev** [comment]
> There's nothing called the ES6 Module Loader. There isn't even an official spec for the module loader yet.

Yeah. English. Sucks. Sorry for misleading. I know about that. 

> Just compile it before testing:

Yeah, but in this case you will load entire library (then you need to test only 1 module form this library). In some cases results can be different (```compile into one file -> test it``` and ```import one module -> test it```). 

I mean, in the end of a day you need to test only 1 specific module -- not the entire library. 

```
library === bundle
```

The approach
```
* import specified module
* test it
* ..
* import specified module
* test it
``` 
pretty much better then
```
* import entire bundle
* test one of the components in this bundle
* test another one
```
I mean, yes, in the perfect world results will be the same. But...

**Comment by sindresorhus** [comment]
I'm sure there's a way to achieve what you want, but I'm not the right person to ask. I've never used JSPM before. I really don't see this as a test runner issue to be honest. Maybe try asking on a JSPM related support forum or something.
